spec/forms/developer_search_form.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeveloperSearchForm do
+  describe '#search' do
+
+    let(:language) { create :language }
+    let(:programming_language) { create :programming_language }
```

Comment:
https://github.com/bbatsov/ruby-style-guide#method-invocation-parens

Reply:
@ntrung2006 
You should write 
```ruby
create(:programming_language)
```
instead of
```ruby
create :programming_language
```
Add parentheses.

But see the following, too.
```ruby
# bad
validates(:name, presence: true)
# good
validates :name, presence: true
```
ref: https://github.com/bbatsov/ruby-style-guide#method-invocation-parens

Reply:
Fixed!